frontend/src/component/executiveDashboard/FlagsProjectChart/FlagsProjectChartComponent.tsx

```diff
@@ -0,0 +1,155 @@
+import { useMemo, type VFC } from 'react';
+import {
+    Chart as ChartJS,
+    CategoryScale,
+    LinearScale,
+    PointElement,
+    LineElement,
+    Title,
+    Tooltip,
+    Legend,
+    TimeScale,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+import 'chartjs-adapter-date-fns';
+import { Paper, Theme, Typography, useTheme } from '@mui/material';
+import {
+    useLocationSettings,
+    type ILocationSettings,
+} from 'hooks/useLocationSettings';
+import { formatDateYMD } from 'utils/formatDate';
+import { ExecutiveSummarySchema } from 'openapi';
+
+const getRandomColor = () => {
```

Comment:
can you add a sample screenshot so we can take a look at the colors that got generated? In other words I'd like to see if the color choice is smart enough to avoid very similar colors